### PR TITLE
add script to create tensorflow serving warmup file

### PR DESCRIPTION
**Pull request by OtherRandomUser**

### CODE REVIEW

File: tools/create_tensorflow_warmup.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+from os import path
+from absl import app, flags
+from absl.flags import FLAGS
+from tensorflow_serving.apis import model_pb2, predict_pb2, prediction_log_pb2
+
+from yolov3_tf2.dataset import preprocess_image, load_tfrecord_dataset
+
+
+flags.DEFINE_string('dataset', None, 'path to the dataset, labels will be ignored')
```
Comment (R-R-Onzi) [inline tools/create_tensorflow_warmup.py]: Estou tentando organizar as FLAGS nossas no padrão abaixo:
flags.DEFINE_string('classes', './data/coco.names',
                    'path to classes file')
flags.DEFINE_string('checkpoints', './out',
                    'folder to save the checkpoints to')

flags.DEFINE_integer('size', 416,
                     'image size')

flags.DEFINE_float('learning_rate', 1e-3,
                   'learning rate')
 
 flags.mark.......
 
 edit: Aqui n ficou legal, melhor olhar https://github.com/retail-intelligence/yolov3-tf2/blob/master/train.py

File: tools/create_tensorflow_warmup.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+def main(argv):
+
+    count = 0
+    images = []
+    files = [path.join(path.join(FLAGS.dataset, f))
+        for f in os.listdir(FLAGS.dataset)
+        if path.isfile(path.join(FLAGS.dataset, f))
+    ]
+
+    files = [f for f in files if f.endswith(('.png', 'jpg', 'jpeg'))]
```
Comment (R-R-Onzi) [inline tools/create_tensorflow_warmup.py]: Não vai ponto no jpg e jpeg?

File: tools/create_tensorflow_warmup.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                )
+            }
+        )
+
+        log = prediction_log_pb2.PredictionLog(
+            predict_log=prediction_log_pb2.PredictLog(request=request)
+        )
+
+        writer.write(log.SerializeToString())
+        print('"tf_serving_warmup_requests" created with success!')
```
Comment (R-R-Onzi) [inline tools/create_tensorflow_warmup.py]: Prefiro usar o logging do absl, mas se não gosta ok.

**Review by R-R-Onzi — CHANGES_REQUESTED** [review]
Só não gostei da oganização das FLAGS fora isso os outros comentarios são minor.

File: tools/create_tensorflow_warmup.py (hunk 3 above)
Comment (tgcandido) [reply tools/create_tensorflow_warmup.py]: seria bom padronizar o absl em todos repos Python?

File: tools/create_tensorflow_warmup.py (hunk 1 above)
Comment (OtherRandomUser) [reply tools/create_tensorflow_warmup.py]: Ok, separei e alinhei as flags em 5f677274bc82b8b87208ac51d6328170812d7c4a

File: tools/create_tensorflow_warmup.py (hunk 2 above)
Comment (OtherRandomUser) [reply tools/create_tensorflow_warmup.py]: typo, corrigido em 5f677274bc82b8b87208ac51d6328170812d7c4a

File: tools/create_tensorflow_warmup.py (hunk 3 above)
Comment (OtherRandomUser) [reply tools/create_tensorflow_warmup.py]: mudei para o logger do absl em 5f677274bc82b8b87208ac51d6328170812d7c4a. eu ahco uma boa padronizar os loggers, entre o absl e o outro logger eu prefiro o absl, só pelo fato dele não precisar de uma grande quantidade de configuração

**Review by R-R-Onzi — APPROVED** [review]

**Review by tgcandido — APPROVED** [review]
